ARO 2021 Conference - scATAC-seq vs Bulk-ATAC-seq

A cellâ€™s regulome consists of all cis-regulatory elements in its genome and encompasses information for gene regulatory networks and transcriptional activities. Many bulk sequencing technologies have been utilized over the years to study the regulome, including DNase-seq, ChIP-seq and ATAC-seq. However, these techniques do not allow for a global understanding of each individual cell population in the tissue of interest. Recent advancements in single-cell technologies have made it possible to study the regulomes of individual cells in complex, heterogeneous tissues such as the cochlea. Single-cell ATAC-seq (scATAC-seq) is the most widely used of these single-cell regulome mapping technologies because of its simple methodology, robust protocols, and scalability to large numbers of cells. Typically, the number of expressed genes or open chromatin sites detected from single cells is lower compared to bulk population-level measurements. The cause for this is multifactorial, including sparse data due to a high frequency of dropout events, transcriptional bursting (temporal fluctuation of transcript levels in individual cells), and higher levels of technical noise. Thus, in choosing techniques for mapping chromatin accessibility in experimental design we are often faced with a dilemma of resolution versus robustness of the chosen methodology. In this study, we compare whole cochlear scATAC-seq to cochlear epithelial (CD326+) and non-epithelial (CD326-) bulk ATAC-seq data for sensitivity and complexity. We also compare three different bioinformatic methods to analyze the scATAC-seq data, including Signac, SnapATAC and scATAC-pro. In addition to global analysis of chromatin accessibility we compare the epigenetic results obtained for several of the canonical inner ear cell type-specific genes. In conclusion, the presented analyses should help investigators in our field choose which methodology and bioinformatic pipeline best suits their experimental design.
